modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -278,55 +278,52 @@ class CV_EXPORTS Platform
     Impl* p;
 };
 
-/*
-//! @brief Attaches OpenCL context to OpenCV
+/**
+// @brief Attaches OpenCL context to OpenCV
```

Comment:
I did notice the extra ```//!``` in the comments, but left it as is, assuming its a opencv coding style.
@mshabunin Thanks for the [example](http://docs.opencv.org/master/d4/db1/tutorial_documentation.html#tutorial_documentation_quick_start_2)